### ISSUE TITLE
FilterNotification is broken

### ISSUE THREAD

**Issue by ferndot** [Lanseuo/onesignal-notifications#3]
Thanks for making this! I have 1 small issue: sending a FilterNotification results in the following:
```
2019-01-12T04:36:26.870607+00:00 app[web.1]: [{'field': 'tag', 'key': 'user_pk', 'relation': '=', 'value': '18cad453-902d-48d2-8726-4337e47fe23b'}]
2019-01-12T04:36:26.872324+00:00 app[web.1]: 'function' object has no attribute 'items'
```

Here is my code. The same content works with a DeviceNotification:
```
notification = FilterNotification(
    filters=[
        Filter.Tag('user_pk', '=', str(user.pk)),
    ],
    **kwargs,
)

# Send notification
print(client.send(notification))
```

**Comment by LucasHild** [comment]
Thanks for reporting this issue.
I fixed the bug in fe0ea1f. It was a problem with Python 2 back-support. I will release a new version to PyPi in the next few hours.

Edit: The new release _v0.2.4_ is now on PyPi. Use `pip install --upgrade onesignal-notifications` for upgrading to the latest version.

**Comment by ferndot** [comment]
Thanks so much for the quick patch!
